components/dash-core-components/src/components/Loading.react.js

```diff
@@ -164,4 +236,41 @@ Loading.propTypes = {
          */
         component_name: PropTypes.string,
     }),
+
+    /**
+     * Setting mode to  "on" or "off"  will override the loading state coming from dash-renderer
+     */
+    mode: PropTypes.oneOf(['auto', 'on', 'off']),
+
+    /**
+     * Add a time delay (in ms) to the spinner being removed to prevent flickering.
+     */
+    delay_hide: PropTypes.number,
+
+    /**
+     * Add a time delay (in ms) to the spinner being shown after the loading_state
+     * is set to True.
+     */
+    delay_show: PropTypes.number,
+
+    /**
+     * Whether the Spinner should show on app start-up before the loading state
+     * has been determined. Default True.  Use when also setting `delay_show`.
+     */
+    show_initially: PropTypes.bool,
+
+    /**
+     * Specify component and prop to trigger showing the loading spinner
+     * example: `[{"output-container": "children"}]`
+     *
+     */
+    target_components: PropTypes.arrayOf(PropTypes.object),
```

Comment:
I wonder if this would be better as
```js
objectOf(arrayOf(string))
```
or perhaps
```js
objectOf(oneOfType([string, arrayOf(string)]))
```
so you could do something like:
```py
target_components={"my_table": ["data", "columns"], "my_dropdown": "options"}
```

Reply:
I added this, plus it's possible to use a wildcard.  This will trigger loading for any prop in the `"my_table"` component
```
target_components={"my_table": "*", "my_dropdown": "options"}
```